oauth/o-auth.md

```diff
@@ -61,29 +61,35 @@ The first thing you want to do is set up your credentials on google by following
 Please make sure that you secure ALL your private keys and sensitive information under your ``.env`` files! </br> 
 #### Step 2 - Connect Google credentials to Auth0 in Connections </br> </br> 
 Log into Auth0, on the left side you will see a menu with a ``Connections`` option. Select ``Connections`` then select the sub category named ``Social`` that will lead you to select the Gmail connection as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png"  width="500" height="500" /></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png" width="500" height="500" /></br>
 
 Once you click on ``Try it!`` you will see a modal pop up where you will have to input the Google credentials you created earlier. Just as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png"  width="500" height="500" /></br></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png" width="500" height="500" /></br></br>
```

Comment:
The images are stored in the folder where other images are stored and I have asked the maintainer to merge my PR.
https://github.com/mmedina3/Auth0-Lesson - The following is the link to the image folder.

Reply:
My PR consists of the images of the code snippets.

Reply:
please move these from mmedina's repo to techtonica's